central_v2/src/lib/CentralModels.tsx

```diff
@@ -64,3 +64,16 @@ export type CreateQuestionLocalData = {
   incompleteCards?: IncorrectCard[] | null,
   completeCards?: IncorrectCard[] | null
 }
+
```

Comment:
different types of fetch based on the screen/library tab the user has navigated to